docs/tasks/configure-pod-container/security-context.md

```diff
@@ -24,6 +24,8 @@ a Pod or Container. Security context settings include:
 
 * [Seccomp](https://en.wikipedia.org/wiki/Seccomp): Limit a process's access to open file descriptors.
 
+* AllowPrivilegeEscalation: Controls whether a process can gain more privileges than it's parent process. This bool directly controls if the [`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) flag will be set on the container process. AllowPrivilegeEscalation is true always when the container is: 1) run as Privileged OR 2) has `CAP_SYS_ADMIN`.
+
```

Comment:
it's -> its

controls if -> controls whether

will be set -> gets set

Reply:
thanks updated!